src/main/java/org/craftercms/studio/impl/v2/upgrade/operations/RenameUpgradeOperation.java

```diff
@@ -24,48 +24,54 @@
 import java.io.File;
 import java.nio.file.Path;
 import java.nio.file.Paths;
-import static org.craftercms.studio.api.v1.util.StudioConfiguration.BLUE_PRINTS_PATH;
 
 /**
  * Implementation of {@link org.craftercms.studio.api.v2.upgrade.UpgradeOperation} that renames the blueprint in the
  * global repository.
  * @author Dejan Brkic
  */
-public class BlueprintRenameFolderUpgradeOperation extends AbstractUpgradeOperation {
+public class RenameUpgradeOperation extends AbstractUpgradeOperation {
 
-    public static final String CONFIG_KEY_BLUEPRINT_FOLDER_NAME = "blueprintFolderName";
-    public static final String CONFIG_KEY_NEW_FOLDER_NAME = "newFolderName";
+    public static final String CONFIG_KEY_SITE = "site";
+    public static final String CONFIG_KEY_OLD_PATH = "oldPath";
+    public static final String CONFIG_KEY_NEW_PATH = "newPath";
 
-    protected String blueprintFolderName;
-    protected String newFolderName;
+    protected String siteId;
```

Comment:
This property is not needed, se the note below

Reply:
I don't want it to be run for all sites